src/components/ContentOpenWith/ContentOpenWith.js

```diff
@@ -286,7 +295,7 @@ class ContentOpenWith extends PureComponent<Props, State> {
                 );
         }
 
-        this.windowRef = null;
+        integrationWindowRef = null;
```

Comment:
Should this be `this.setState({ integrationWindowRef: null })`?

Reply:
I want to set the actual reference to null here for cleanup purposes. I'm not sure if clearing the actual state here could wipe our loading UI early..